.gitlab-ci.yml

```diff
@@ -196,6 +196,23 @@ package-arm:
   variables:
     ARCH: arm64
 
+package-oci:
+  stage: package
+  extends: .package-oci
+  rules:
+    - if: $DOTNET_PACKAGE_VERSION
+      when: on_success
+    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/'
+      when: manual
+      allow_failure: false
+  script:
+    - ../.gitlab/build-oci.sh
+  parallel:
+    matrix:
+      - ARCH:
+        - arm64
+        - amd64
```

Comment:
We need to support `musl` too - not sure how you need to handle that here though 😄

Reply:
Do we? i don't think musl is supported by the auto injecter anyway

Reply:
I don't think the agent (resp. updater) can run on a host with `musl`. How is `musl` used today for `deb`s & `rpm`s?

We can package it anyway though, there are ways to add "OS Features" to the platform selection. Happy to implement support for that in the packaging tool if needed 😄

Reply:
> Do we? i don't think musl is supported by the auto injecter anyway

Oh, really 🤔 I guess for host injection that may be true (I don't know), we do support it for k8s lib-injection. 

> How is musl used today for debs & rpms?

Currently we only produce musl tar.gz files 🙂 

> We can package it anyway though, there are ways to add "OS Features" to the platform selection

Interesting, I suspect we _should_ add that to our existing deb/rpms then!